bumblefoot/src/data.rs

```diff
@@ -2,12 +2,18 @@
 // use anyhow::Result as AnyResult;
 use serde_derive::{Deserialize, Serialize};
 use std::collections::HashMap;
+extern crate exitcode;
```

Comment:
does this compile for you?
```suggestion
use exitcode;
```

Reply:
Yes, without any problems.
why are you asking?